api/database/sampleApp.py

```diff
@@ -0,0 +1,68 @@
+from flask import Flask, request, jsonify
+from mongoengine import *
+from pymongo.errors import DuplicateKeyError
+from users import Users
+
+app = Flask(__name__)
+
+@app.route('/users', methods = ["GET"])
+def get_users():
+    name = request.args.get('Name')
+    email = request.args.get('Email')
+    password = request.args.get('Password')
+
+    data = []
+    for user in Users.objects:
+        if name is not None:
+            if user.name == name:
+                data.append(user.to_json())
+            else:
+                pass
+        elif email is not None:
+            if user.email == email:
+                data.append(user.to_json())
+            else:
+                pass
+        else:
+            data.append(user.to_json())
+    
+    dataDict = {'users': data}
+    
+    return dataDict, 200
```

Comment:
what does this 200 represent?

Reply:
It specifies the status of the api call (200 = ok). This is for communication to the website once that is hooked up.
Look at this for the different status codes: https://httpstatuses.com/